CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # ==================================================================================================
 # CMake
 # ==================================================================================================
-cmake_minimum_required(VERSION 3.19)
+cmake_minimum_required(VERSION 3.16.5)
```

Comment:
Don't lower the CMake requirements

Reply:
I will revert.  Is there a feature currently implemented that's dependent on 3.19?

Yocto dunfell currently has CMake at 3.16.5.  I will need to create bbappend recipe to build with newer CMake, and that will be a dependency for filament recipe (klunky).